src/utils/schema-preprocessor.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Swagger 2.0 doesn't support multiple types, while JSON-Schema does. This method adds null as supported type for all non-required attributes
+ * @param {Object} bodySchema
+ */
+function makeOptionalAttributesNullable(bodySchema) {
+    if (!bodySchema || bodySchema.length === 0) {
+        return;
+    }
+
+    for (let schemaEntry of bodySchema) {
+        if (schemaEntry && schemaEntry.schema) {
+            _processSchemaEntry(schemaEntry.schema);
+        }
+    }
+}
+
+function _processSchemaEntry(schema) {
+    // if it is array, process schema of its entries
+    if (!schema.properties && schema.items) {
+        schema = schema.items;
+    }
+
+    if (!schema.properties) {
+        return;
```

Comment:
Please at lease cover this case with test, I get the idea why it's hard to cover all the branches that just there to protect from accessing `undefined` but this one seems to be different

Reply:
Sure!

Reply:
@idanto Done.